tests/api/users.spec.js

```diff
@@ -0,0 +1,39 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const jwt = require('jsonwebtoken')
+const server = require('../../app')
+const User = require('../../models/user')
+
+const { expect } = chai
+
+chai.use(chaiHttp)
+
+describe('GET /', () => {
+  beforeEach(() => {
+    // const user = User.create({
+    //   username: 'tung',
+    //   email: 'tung@teng.com',
+    //   password: 'abcxyz'
+    // })
+  })
+
+  it('should return status 401 if no token provided', (done) => {
+    chai.request(server)
+      .get('/api/users')
+      .end((err, res) => {
+        expect(res).to.have.status(401)
```

Comment:
I think 4xx and 5xx status codes are considered as errors, so you should use `err`, not `res`